test/functional/wallet_send.py

```
@@ -460,5 +460,45 @@ def run_test(self):
         assert res["complete"]
 
 
+        dec = self.nodes[0].decodepsbt(signed["psbt"])
```

Ruff (0.12.2)

Undefined name `signed`

(F821)